pymc/backends/mcbackend.py

```diff
@@ -201,6 +201,42 @@ def _slice(self, idx: slice) -> "IBaseTrace":
     def point(self, idx: int) -> dict[str, np.ndarray]:
         return self._chain.get_draws_at(idx, [v.name for v in self._chain.variables.values()])
 
+    def completed_draws_and_divergences(self, chain_specific: bool = True) -> tuple[int, int]:
```

Comment:
question why do we have special handling for divergences, instead of being sampler agnostic?